spec/tapioca/compilers/symbol_table_compiler_spec.rb

```diff
@@ -531,6 +531,52 @@ class WebMockHTTPClient
       assert_equal(output, compile)
     end
 
+    it("compiles a constant which is aliased to a constant that has been overwritten as a placeholder") do
+      add_ruby_file("bar.rb", <<~RUBY)
```

Comment:
Super nitpicky, but can we give this "file" a better name? Maybe:
```suggestion
      add_ruby_file("overwritten_class_module_references.rb", <<~RUBY)
```
or something